content/data-package/contents.lr

```diff
@@ -310,3 +310,10 @@ The datetime must conform to the string formats for datetime as described in [RF
 [dc-temporal]: http://dublincore.org/documents/usageguide/qualifiers.shtml#temporal
 [markdown]: http://commonmark.org/
 
+## Data Package Profiles
```

Comment:
@pwalsh I'm not clear on relation of this to http://specs.frictionlessdata.io/profiles/

At the moment what we have here falls a bit between two stools: on the one hand, not substantive enough to explain fully to users what this is and why it is important; on the other hand not a simple: here is this property and see the /profiles/ page.

My inclination would be to reference the profiles "spec" and just define the existence of the property and the fact it has a default value.